.github/workflows/pr-title-lint.yml

```diff
@@ -0,0 +1,24 @@
+name: pr-title-lint
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - synchronize
+      - edited
+
+jobs:
+  pr-title-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v3
+
+      - name: Setup commitlint
+        run: |
```

Comment:
**NIT**:

What do you think this particular step is added to the https://github.com/elastic/synthetics/blob/main/package.json ?

This could help with running the same command locally and in the CI.

Reply:
I was thinking about this too.

I had the assumption that a developer won't we able to lint the PR title locally anyway.

But you are right, it would still make it more debuggable.

Reply:
https://github.com/elastic/synthetics/pull/828/commits/626208f353fbfde9baf431714c5a21264f627601